docs/swagger/spec.yml

```diff
@@ -2113,6 +2113,19 @@ parameters:
       - ABORTED
 
 definitions:
+  RenderDefinition:
+    type: object
+    properties:
+      breakpoints:
+        type: object
+        description: mapping from a tile cell value to associated color
+      scale:
+        type: string
+        description: The type of scale to use (e.g. 'continuous', 'qualitative', 'diverging')
+        note: qualitative strings will contain a hex color (e.g. 'qualitative[#ff00ffff]')
```

Comment:
it's complaining about this - make this a part of the description since `note` isn't a valid property on it